Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'discard' # Soft deletes for ActiveRecord
 gem 'sidekiq' # background processing for Ruby
 gem 'bootsnap', require: false # Reduces boot times through caching; required in config/boot.rb
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby] # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'faraday' # HTTP client
+gem 'nokogiri' # Gem to work with XML and HTML files
```

Comment:
The descriptions can be improved slightly:

```suggestion
gem 'farday' # HTTP client library, with support for multiple backends
gem 'nokogiri' # Work with XML and HTML from Ruby
```

> **Note**
> In general, you can take the short description on the repository
> <img width="1294" alt="image" src="https://github.com/Goose97/google-scraper-ruby/assets/696529/0aae9a97-6c8e-4dd0-9254-822dfdf150a0">

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/802acb5ec77fc85f179ae4c144dd321054e4e1a5